### ISSUE TITLE
Update the Installing OpenShift on a single node instructions in the end user docs

### ISSUE THREAD

**Issue by mburke5678** [okd-project/planning#27]
https://github.com/openshift/openshift-docs/issues/51152

The OKD end user docs have instructions for Installing "OpenShift on a single node" that is OCP-specific.

It needs to be significantly rewritten:

- users will need to use installer from github releases instead of mirror.openshift.com
- references to console.redhat.com need to be replaced with selfhosted assisted installer

The files are to be hidden on OKD docs until we can make these changes. 

Assemblies:
installing/installing_sno/install-sno-preparing-to-install-sno.adoc
installing/installing_sno/install-sno-installing-sno.adoc

**Comment by binnes** [comment]
Merge with #2

**Comment by JM1** [comment]
There are two ways to install SNO: 
* using Agent-based Installer 
* the older (i think) approach which is [documented in the OCP docs](https://docs.openshift.com/container-platform/4.13/installing/installing_sno/install-sno-preparing-to-install-sno.html)

The Agent-based Installer approach will work when [this has been merged](https://github.com/openshift/installer/pull/6619). The OCP-documented approach works in 4.15 and will work in [4.14 when this has been merged](https://github.com/openshift/installer/pull/7479).

**Comment by blaisep** [comment]
FWIW, I'm looking forward to this because I have a server in my homelab waiting for OKD.

**Comment by peterroth** [comment]
Good news:
```
~: oc get nodes
NAME           STATUS   ROLES                         AGE   VERSION
okd-sno-test   Ready    control-plane,master,worker   46m   v1.27.9+e36e183
~: oc get clusterversion
NAME      VERSION                          AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.14.0-0.okd-2024-01-26-175629   True        False         25m     Cluster version is 4.14.0-0.okd-2024-01-26-175629
```

Now just need to document it.

**Comment by peterroth** [comment]
Hello @mburke5678,
I started to document the required steps and got a question: as I saw in a [previous commit you had](https://github.com/openshift/openshift-docs/pull/70149/commits/3fdeb2354891bf66bd40768bd2763dd32c16ada8), it's possible to mark if a part is only for OCP or if that's for OKD. Is it necessary to add `ifndef::openshift-origin[]` in case the given section is only for OCP?
My feeling is that it's required, but I'd like to be sure.
